.babelrc

```diff
@@ -1,11 +1,20 @@
 {
 	"presets": [
-		"es2015",
+		[ "env", {
+			"targets": {
+				"browsers": [
+					"last 2 versions",
+					"Safari >= 9",
+					"iOS >= 9",
+					"not ie <= 10",
+					"> 1%"
+				]
+			}
+		} ],
 		"stage-2"
 	],
 	"plugins": [
 		"transform-runtime",
-		"transform-class-properties",
```

Comment:
Can we drop any of the other plugins? And remind me @aduth why aren't we just using the `react` preset anyway? 😄